### PR TITLE
Update library path

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -450,6 +452,9 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
         COMMAND ln -sf libopencv_imgcodecs.so libopencv_imgcodecs.so.${OPENCV_VERSION}
```
Comment (Tabrizian) [inline CMakeLists.txt:452]: @mc-nv Can you remind me which torch library required these libraries?

File: CMakeLists.txt [hunk 2]
```diff
@@ -440,7 +442,7 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
     )
   ENDFOREACH(plib)
 
-  set(OPENCV_VERSION "4.6.0")
+  set(OPENCV_VERSION "406")
```
Comment (Tabrizian) [inline CMakeLists.txt:445]: In the new container, I can see that both "406" and 4.6.0" are available. Given that we have downgraded mkl version, can be just use 4.6.0 instead?

File: CMakeLists.txt (hunk 1 above)
Comment (mc-nv) [reply CMakeLists.txt:456]: This lib is required by `libopencv_videoio.so`.

File: CMakeLists.txt (hunk 2 above)
Comment (mc-nv) [reply CMakeLists.txt:449]: I got that value using the following command `ldd *so* | grep -E ':|libopencv_'` against `pytrorch_backend` 
And LDD was not able to find shared object dependencies unless values was updated.
```
libopencv_calib3d.so:
        libopencv_features2d.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_features2d.so.406 (0x00007f01ca4e8000)
        libopencv_flann.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_flann.so.406 (0x00007f01bf7a0000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f01bf0f5000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f01becb7000)
libopencv_calib3d.so.406:
        libopencv_features2d.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_features2d.so.406 (0x00007fcc6cb53000)
        libopencv_flann.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_flann.so.406 (0x00007fcc61e0b000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007fcc61760000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007fcc61322000)
libopencv_core.so:
libopencv_core.so.406:
libopencv_features2d.so:
        libopencv_flann.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_flann.so.406 (0x00007f3363382000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f3362cd7000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f3362899000)
libopencv_features2d.so.406:
        libopencv_flann.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_flann.so.406 (0x00007f595fcb7000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f595f60c000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f595f1ce000)
libopencv_flann.so:
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f4e05e35000)
libopencv_flann.so.406:
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007fdab5b72000)
libopencv_highgui.so:
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f00df3af000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f00def71000)
libopencv_highgui.so.406:
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f674465c000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f674421e000)
libopencv_imgcodecs.so:
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f6cb44ca000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f6cb3f81000)
libopencv_imgcodecs.so.406:
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f40827a4000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f408225b000)
libopencv_imgproc.so:
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f9901f28000)
libopencv_imgproc.so.406:
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f5e781c3000)
libopencv_video.so:
        libopencv_calib3d.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_calib3d.so.406 (0x00007f2cea414000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f2ce9d69000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f2ce992b000)
        libopencv_features2d.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_features2d.so.406 (0x00007f2ce9306000)
        libopencv_flann.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_flann.so.406 (0x00007f2cde716000)
libopencv_video.so.406:
        libopencv_calib3d.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_calib3d.so.406 (0x00007f7db9e45000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f7db979a000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f7db935c000)
        libopencv_features2d.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_features2d.so.406 (0x00007f7db8d37000)
        libopencv_flann.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_flann.so.406 (0x00007f7dae147000)
libopencv_videoio.so:
        libopencv_imgcodecs.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgcodecs.so.406 (0x00007fd2b8fe7000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007fd2b8910000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007fd2b84d2000)
libopencv_videoio.so.406:
        libopencv_imgcodecs.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgcodecs.so.406 (0x00007f61ca954000)
        libopencv_imgproc.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_imgproc.so.406 (0x00007f61ca27d000)
        libopencv_core.so.406 => /opt/tritonserver/backends/pytorch/./libopencv_core.so.406 (0x00007f61c9e3f000)
```

**Review by Tabrizian — APPROVED** [review]
Approving for now but there are still additional changes required that I'll include in a separate PR.

**Review by Tabrizian — APPROVED** [review]

**Pull request merged**